### PR TITLE
Check for the presence of Library and program

### PR DESCRIPTION
**Pull request by janus**
Check for the presence of  efence Library and emcc program

### CODE REVIEW

File: b.sh [hunk 1]
```diff
@@ -19,10 +19,24 @@ if [ -e "$OUTPUT_FILE" ]; then
 	rm "$OUTPUT_FILE"
 fi
 
+
+ldconfig -p | grep efence  &>/dev/null
+if ! [ $? -eq 0 ]; then
+        echo "efence libraray is missing"
```
Comment (jbampton) [inline b.sh:25]:
```suggestion
        echo "efence library is missing"
```

**Review by jbampton — CHANGES_REQUESTED** [review]
We are using tabs for indents in shell files.

Use `shfmt` to reformat etc 

https://github.com/mvdan/sh

See our default styles in the `.editorconfig` file

https://github.com/SalamLang/Salam/blob/8ffe13ed4a51a0af75856784dd2c74b906aef07e/.editorconfig#L18

**Comment by janus** [comment]
> We are using tabs for indents in shell files.
> 
> Use `shfmt` to reformat etc
> 
> https://github.com/mvdan/sh
> 
> See our default styles in the `.editorconfig` file
> 
> https://github.com/SalamLang/Salam/blob/8ffe13ed4a51a0af75856784dd2c74b906aef07e/.editorconfig#L18

Alright

**Comment by erfangnu** [comment]
merged.
thanks, let's push faster

**Pull request merged**